Gemfile

```diff
@@ -1,15 +1,15 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.2.1"
+ruby '3.2.1'
```

Comment:
I specially left `"` everywhere here to reduce amount of unnecessary changes.
Later it will be easier to update file when a new release of rails is published.